communication/moderation.md

```diff
@@ -1,38 +1,58 @@
-# Moderation on Kubernetes Communications Channels
+# Moderation on Kubernetes communications channels
```

Comment:
going on record that I don't like not capitalizing title names of documents. i will put in a separate PR to the style guide.

Reply:
Updated to use Title Case. When you PR the style guide, I would link out to a doc that describes title case well.